resources/test_data/tbl/meta_tables/meta_segments.tbl

```diff
@@ -1,9 +1,9 @@
 table_name|chunk_id|column_id|column_name|column_data_type|encoding_type|vector_compression_type|estimated_size_in_bytes
 string|int|int|string|string|string_null|string_null|long
-int_int|0|0|a|int|null|null|176
-int_int|0|1|b|int|null|null|176
-int_int|1|0|a|int|null|null|172
-int_int|1|1|b|int|null|null|172
-int_int_int_null|0|0|a|int|RunLength|null|96
-int_int_int_null|0|1|b|int|Dictionary|SimdBp128|100
-int_int_int_null|0|2|c|int|null|null|265
\ No newline at end of file
+int_int|0|0|a|int|null|null|240
+int_int|0|1|b|int|null|null|240
+int_int|1|0|a|int|null|null|236
+int_int|1|1|b|int|null|null|236
+int_int_int_null|0|0|a|int|RunLength|null|160
+int_int_int_null|0|1|b|int|Dictionary|SimdBp128|164
+int_int_int_null|0|2|c|int|null|null|329
```

Comment:
Where do these changes come from? Is #1968 not merged yet? Could you merge the current master?

Reply:
The tests failed after having implemented the access counters. The expected values were 64 bytes off. Exactly the size of the access counter structure.

Reply:
Alright!